src/test/regress/expected/multi_extension.out

```diff
@@ -492,30 +492,17 @@ SELECT * FROM multi_extension.print_extension_changes();
 
 -- Test downgrade to 9.4-1 from 9.5-1
 ALTER EXTENSION citus UPDATE TO '9.5-1';
-BEGIN;
-  SET citus.enable_metadata_sync TO on;
-  SELECT master_add_node('localhost', :master_port, groupId=>0);
-NOTICE:  localhost:xxxxx is the coordinator and already contains metadata, skipping syncing the metadata
- master_add_node
----------------------------------------------------------------------
-               1
-(1 row)
-
-  CREATE TABLE citus_local_table (a int);
-  SELECT create_citus_local_table('citus_local_table');
-NOTICE:  create_citus_local_table is deprecated in favour of citus_add_local_table_to_metadata
- create_citus_local_table
----------------------------------------------------------------------
-
-(1 row)
-
-  RESET citus.enable_metadata_sync;
-  -- downgrade from 9.5-1 to 9.4-1 should fail as we have a citus local table
-  ALTER EXTENSION citus UPDATE TO '9.4-1';
-ERROR:  citus local tables are introduced in Citus 9.5
-HINT:  To downgrade Citus to an older version, you should first convert each citus local table to a postgres table by executing SELECT undistribute_table("%s")
-CONTEXT:  PL/pgSQL function inline_code_block line XX at RAISE
-ROLLBACK;
+-- TODO: This test should be moved to a valid downgrade testing suite where the downgrade is done, both on the schema and the binaries. Later changes in Citus made a C vs Schema discrepancy error here
+-- BEGIN;
+--   SET citus.enable_metadata_sync TO on;
+--   SELECT master_add_node('localhost', :master_port, groupId=>0);
+--   CREATE TABLE citus_local_table (a int);
+--   SELECT create_citus_local_table('citus_local_table');
+--   RESET citus.enable_metadata_sync;
+--
+--   -- downgrade from 9.5-1 to 9.4-1 should fail as we have a citus local table
+--   ALTER EXTENSION citus UPDATE TO '9.4-1';
+-- ROLLBACK;
```

Comment:
This test runs into schema vs binary incompatibility. 

Have disabled it for now, would be great to have more robust downgrade tests where binary and schema actually do conform.